app/models/user.rb

```diff
@@ -6,9 +6,7 @@ class User < ApplicationRecord
   validates :email, email: { strict_mode: true }, if: "uid.nil?", on: :create
 
   enum role: %w(customer venue_admin platform_admin)
-
-  after_create :send_welcome_email
```

Comment:
@matthewrpacker Love that you're taking this out of the `User` model ✨ 

Does the `send_welcome_email` method still exist in here? If so, we should probably get rid of it.